package com.example.todoapp.exceptions

class PopulateNewTaskException: Exception()